File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        
        for (int i=0; i<nums.size() - 1; i++) {
            if (nums[i] == val) {
                nums[i].delete;
                val++;
            }
        }

        return k;
    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        res = []
        i, j = 0, 0

        for i in range(len(nums)):
            n = 1

            for j in range(len(nums)):
                if j == i:
                    continue
                n *= nums[j]

            res.append(n)

        return res
